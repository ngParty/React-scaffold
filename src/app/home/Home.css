:root {
  --mainColor: tomato;
}

.header {
  color: var(--mainColor);
  padding: 1em;
}

.my-logo {
  color: var(--mainColor);
  width: 100px;
  height: 100px;
  padding: 10px;
  background: url("../../assets/img/ngPartyCZ_01.png") no-repeat;
  background-size: contain;
}
