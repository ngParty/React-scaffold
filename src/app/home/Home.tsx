import * as React from 'react';

import * as styles from './Home.css';
import img from '../../assets/img/ngPartyCZ_01.png';
import svg from '../../assets/img/ngPartyCz_02.svg';
const imgByAssetsPath = '../../assets/img/ngPartyCZ_01.png';

const Home = () => {
  return (
    <div>
      <h2 className={styles.header}>Home</h2>
      <div>Hello from Sofia!</div>
      <section>
        <div className={styles.myLogo} />
        <img width={150} src={svg} alt="ngParty logo via SVG"/>
        <img width={150} src={img} alt="ngParty logo via IMG"/>
        <img  width={250} src={imgByAssetsPath} alt="ngParty Logo via Path"/>
      </section>
    </div>
  );
};

export default Home;
