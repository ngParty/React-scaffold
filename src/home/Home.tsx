import * as React from 'react';

import * as styles from './Home.css';

const Home = () => {
  return (
    <div>
      <h2 className={styles.header}>Home</h2>
      <div>Hello from Sofia!</div>
    </div>
  );
};

export default Home;
