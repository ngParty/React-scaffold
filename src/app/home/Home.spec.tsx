import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Home from './Home';

test( `should render <Home/>`, () => {
  const component = renderer.create(
    <Home/>
  );

  let tree = component.toJSON();
  expect( tree ).toMatchSnapshot();
} );
