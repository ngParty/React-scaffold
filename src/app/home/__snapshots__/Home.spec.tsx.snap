exports[`test should render <Home/> 1`] = `
<div>
  <h2
    className="header">
    Home
  </h2>
  <div>
    Hello from Sofia!
  </div>
  <section>
    <div
      className="myLogo" />
    <img
      alt="ngParty logo via SVG"
      src={undefined}
      width={150} />
    <img
      alt="ngParty logo via IMG"
      src={undefined}
      width={150} />
    <img
      alt="ngParty Logo via Path"
      src="../../assets/img/ngPartyCZ_01.png"
      width={250} />
  </section>
</div>
`;
